### PR TITLE
Tweaking apa-cv to accord with discussion in https://github.com/citat…

### PR DESCRIPTION
**Pull request by reagle**
Here's the change to apa-cv.csl that I would like to use as discussed in #1619 . This renders the complete reference in place of the cite.

### CODE REVIEW

**Comment by reagle** [comment]
Someone's going to have to help me with the failed checks and what that means.

**Comment by rmzelle** [comment]
There are errors for two things:

* Repository styles may not contain any unused macros. The offending macros are "author-short", "citation-locator", and "issued-year", so these need to be deleted.
* `<citation hanging-indent="true" et-al-min="8" et-al-use-first="6" et-al-use-last="true" entry-spacing="0" line-spacing="2">` uses a number of attributes that are only allowed on `<bibliography>`: "hanging-indent", "entry-spacing", and "line-spacing".

File: apa-cv.csl [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
   <info>
-    <title>American Psychological Association 6th edition (curriculum vitae, sorted by descending date)</title>
+    <title>American Psychological Association 6th edition (curriculum vitae, citation becomes reference, no bibliography)</title>
     <title-short>APA</title-short>
     <id>http://www.zotero.org/styles/apa-cv</id>
     <link href="http://www.zotero.org/styles/apa-cv" rel="self"/>
     <link href="http://owl.english.purdue.edu/owl/resource/560/01/" rel="documentation"/>
-    <author>
+    <autho>
```
Comment (rmzelle) [inline apa-cv.csl]: @reagle, `<autho>` should read `<author>`. I always recommend running the style through https://validator.citationstyles.org/ after hand-editing to find typos like this.

**Comment by reagle** [comment]
Okay, passed automated tests.

**Comment by adam3smith** [comment]
Is there a reason you've removed the bibliography here? What I suggested on the ticket was to duplicate bibliography and citation. Removing the bibliography won't work e.g. for how the style is used on zotero.org -- but I also think generally the bibliography is the expected place for, well, the bibliography.

**Comment by reagle** [comment]
@adam3smith, okay, I will restore the bibliography, which will then be redundant with the inline citations. Maybe there'll be a way to suppress them in pandoc.

**Comment by reagle** [comment]
Okay, uploaded and it passed the tests. `suppress-bibliography: true` works with pandoc.

**Comment by adam3smith** [comment]
excellent -- I was hoping there'd be a way to suppress the bib in Pandoc. Will review tonight & merge

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**